### ISSUE TITLE
the download returned a result different to 200(OK): 404

### ISSUE THREAD

**Issue by GamaqueEc** [V10lator/NUSspli#162]
I have been trying to download games on Nusspli for several days, but every time, I get an error. 
The error says: the download returned a result different to 200(OK): 404 even though I try to download super light games like Super Mario Bros. 3 or even Nes remix.
I've been uninstalling, installing, repairing Nusspli but I can't get it to download any game. I download them on the USB and nothing, on the NAND and I can't install or download anything via Nusspli. can anyone help me?

**Comment by Onomastik** [comment]
Same issue here, with different setup (downloading to SD and installing to USB, downloading only on SD).

**Comment by WaluigisCousin** [comment]
I've been having the same issue. Only has been happening since today but I haven't tried the past 2 days so maybe it could have stayed then as well.

**Comment by zazFe** [comment]
yea having the same issue here as well don't have any clue why.

**Comment by Masamune3210** [comment]
Can you try downloading something from the eshop through the console and make sure that it can properly download a title?

**Comment by zazFe** [comment]
> Can you try downloading something from the eshop through the console and make sure that it can properly download a title?

weirdly when I open the eshop it says its under maintenance I don't know if that's because of the custom firmware though or not then again I'm pretty new to modding in general so I would have no clue. I am assuming though that Nintendo is actually going through legitimate maintenance on the eshop. so this might answer the question, but as I said before I have no clue.

**Comment by Onomastik** [comment]
I tried downloading through the eShop and I got the same issue @zazFe had.

I have to mention that even if I got the message "the download returned a result different to 200(OK): 404" and still get it right now when I use Nusspli, I was able to download and install one small title while getting this message during the process.


________________________________
De : zazFe ***@***.***>
Envoyé : Tuesday, November 8, 2022 6:02:03 AM
À : V10lator/NUSspli ***@***.***>
Cc : Onomastik ***@***.***>; Comment ***@***.***>
Objet : Re: [V10lator/NUSspli] the download returned a result different to 200(OK): 404 (Issue #162)


Can you try downloading something from the eshop through the console and make sure that it can properly download a title?

weirdly when I open the eshop it says its under maintenance I don't know if that's because of the custom firmware though or not then again I'm pretty new to modding in general so I would have no clue. I am assuming though that Nintendo is actually going through legitimate maintenance on the eshop. so this might answer the question, but as I said before I have no clue.

**Comment by datboi78** [comment]
I wanted to comment on this, as I believe the problem is not in the NusSpli itself, rather Nintendo doing regular maintence for their e-shop at the moment. According to perfectly nintendo they should continue through today as well depending on the region

”The maintenance will take place on the following day(s) and at the following time:

Europe: November 7th (4PM) to November 8th (7AM)
UK: November 7th (3PM) to November 8th (6AM)
North America (ET): November 7th (10AM) to November 8th (1AM)
North America (PT): November 7th (7AM to 10PM)
Japan: November 8th (12AM to 3PM)”

Considering NusSpli provides every region, it would be wise to not expect results before all of the maintenance is finished.

Source: https://www.perfectly-nintendo.com/nintendo-maintenance-page/

**Comment by V10lator** [comment]
> when I open the eshop it says its under maintenance

And that's the issue. Please wait untill the maintainance on Nintendos site is over.

**Comment by Onomastik** [comment]
On my side, I tried a different setup with Nusspli (save on SD, install on USB, keep the files downloaded) and it worked like a charm.

I think that the error comes from downloading and installing a game to two different places, while asking Nusspli to delete the files once the game is installed.

TL;DR : Save SD/Install USB/Deletion after installation didn't work and gave the error 404, but Save SD/Install USB/Keep the files after installation works perfectly.
________________________________
De : V10lator ***@***.***>
Envoyé : Tuesday, November 8, 2022 9:40:53 AM
À : V10lator/NUSspli ***@***.***>
Cc : Onomastik ***@***.***>; Comment ***@***.***>
Objet : Re: [V10lator/NUSspli] the download returned a result different to 200(OK): 404 (Issue #162)


when I open the eshop it says its under maintenance

And that's the issue. Please wait untill the maintainance on Nintendos site is over.

**Comment by V10lator** [comment]
@Onomastik The maintainance probably just ended in your region in between the two download tries. Here are the estimated maintainance times:

- Europe: November 7th (4PM) to November 8th (7AM)
- UK: November 7th (3PM) to November 8th (6AM)
- North America (ET): November 7th (10AM) to November 8th (1AM)
- North America (PT): November 7th (7AM to 10PM)
- Japan: November 8th (12AM to 3PM)

(Source: https://www.perfectly-nintendo.com/nintendo-maintenance-page/ )

**Comment by Masamune3210** [comment]
This is why I said to try downloading something. The app pulls from the same servers the eshop does, so if the eshop isn't working, nusspli isn't going to either. Its a good test to make sure that you have a sound base to figure anything else out should eshop not be having issues
